src/app.module.ts

```diff
@@ -1,22 +1,19 @@
 import { Module } from "@nestjs/common";
 import { AuthModule } from "src/modules/auth/auth.module";
-import { UserModule } from "src/modules/user/user.module";
 import { TypeOrmModule } from "@nestjs/typeorm";
-import { UserRecord } from "src/core/infrastructure/entities/User";
 import { UserSessionRecord } from "src/core/infrastructure/entities/Auth";
 
 @Module({
 	imports: [
 		AuthModule,
-		UserModule,
 		TypeOrmModule.forRoot({
 			type: "postgres",
 			host: "localhost",
 			port: 5432,
 			username: "postgres",
 			password: "password",
 			database: "tribal",
-			entities: [UserRecord, UserSessionRecord],
```

Comment:
Why would you remove UserRecord? We need this in the App. All entities must be defined here if they exist in the project

Reply:
Basically if an Entity exists on the project. Keep it here. Never remove entities after they've been added